flow/flow_ebos_energy.cpp

```diff
@@ -62,8 +62,8 @@ int flowEbosEnergyMain(int argc, char** argv)
     Dune::MPIHelper::instance(argc, argv).rank();
 #endif
 
-    Opm::FlowMainEbos<TTAG(EclFlowEnergyProblem)> mainfunc;
-    return mainfunc.execute(argc, argv);
+    Opm::FlowMainEbos<TTAG(EclFlowProblem)> mainfunc;
```

Comment:
You changed the typetags here. A little aggressive copy-paste... Also for foam, two-phase etc. I'll check better the next time!

Reply:
Ok thanks!